canbench-bin/tests/expected/reports_repeated_scope_in_existing_benchmark.txt

```diff
@@ -2,22 +2,22 @@
 
 Benchmark: bench_repeated_scope_exists
   total:
-    instructions: 16.48 K (regressed from 0)
+    instructions: 14.96 K (regressed from 0)
     heap_increase: 0 pages (no change)
     stable_memory_increase: 0 pages (no change)
 
-  scope_1 (scope):
-    instructions: 8124 (regressed by 915.50%)
-    heap_increase: 0 pages (improved by 100.00%)
-    stable_memory_increase: 0 pages (no change)
+  something (scope):
+    instructions: 8124 (new)
```

Comment:
You will need to update things here accordingly if you change the names of scopes as we have an inconsistency now. The benchmark is supposed to be checking an existing scope (as hinted by the benchmark name `bench_repeated_scope_exists`) but instead we consider it new (which is covered by another benchmark).